packages/console/src/cloud/AppRoutes.tsx

```diff
@@ -13,18 +13,19 @@ import SocialDemoCallback from './pages/SocialDemoCallback';
 
 /** Renders necessary routes when the user is not in a tenant context. */
 function AppRoutes() {
+  const profileRoutes = useRoutes(profile);
+
   return (
     <div className={styles.app}>
       <Routes>
         <Route path={GlobalAnonymousRoute.Callback} element={<Callback />} />
         <Route path={GlobalAnonymousRoute.SocialDemoCallback} element={<SocialDemoCallback />} />
         <Route element={<ProtectedRoutes />}>
-          {isCloud && (
-            <Route
-              path={`${GlobalRoute.AcceptInvitation}/:invitationId`}
-              element={<AcceptInvitation />}
-            />
-          )}
+          <Route
```

Comment:
Want to make sure this feature is available in OSS.

Reply:
This whole `/cloud/AppRoutes.tsx` is only rendered in Cloud env, so no need to add `isCloud` inside